sys/include/net/gnrc/rpl.h

```diff
@@ -82,6 +82,17 @@
  *   CFLAGS += -DCONFIG_GNRC_RPL_WITHOUT_VALIDATION
  *   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  *
+ * TODO
+ * ------
+ *
+ * The GNRC RPL implementation only implements storing mode with OF0 ([RFC6552](https://tools.ietf.org/html/rfc6552)).
```

Comment:
Maybe also add the sentence

```suggestion
 * The GNRC RPL implementation only implements storing mode with OF0 ([RFC6552](https://tools.ietf.org/html/rfc6552)).
 * The RPL routing header is parsed by the nodes, but anything else for non-storing mode is missing.
```

Maybe the more appropriate place would be with `Non-storing mode` below. Not sure...